serve/graph/model/models_gen.go

```diff
@@ -342,61 +365,14 @@ type UnexpectedMessage struct {
 func (UnexpectedMessage) IsMessageValue() {}
 
 // `UnknownEvent` is an unknown event type.
-// It has `key` and `value`.
+// It has `value`.
 type UnknownEvent struct {
-	// `value` is an event string..
+	// `value` is an raw event string.
```

Comment:
```suggestion
	// `value` is a raw event string.
```

Reply:
Fixed in this commit. https://github.com/gnolang/tx-indexer/pull/56/commits/bbc5645ab9950e40ed89cc73e5fb3faeab5b75ac